README.md

```diff
@@ -1,20 +1,94 @@
 [![Build Status](https://api.travis-ci.org/byu-dml/metalearn.png)](https://travis-ci.org/byu-dml/metalearn)
 
-# metalearn
-The Data Mining Lab of the Computer Science Department of Brigham Young
-University (BYU-DML) python library of meta-learning tools.
+# Metalearn
 
-Currently extracts meta-features from tabular datasets with categorical
-targets.
+The Data Mining Lab of the Computer Science Department of Brigham Young University (BYU-DML) python3 library of meta-learning tools.
+Extracts **general**, **statistical**, **information-theoretic**, **landmarking** and **model-based** meta-features from tabular datasets for use in meta-learning applications.
 
-In order for the test suite to run properly, our package requires Python3.6
+## Installation
 
-You can install the requirements for this package using:
-pip install -r requirements.txt
+Dependencies installed with this package include:
+- pandas
+- numpy
+- scikit-learn
 
-The test suite has one required package that is not available on pypi.
-This package, openml, should be accessed by git cloning the following
-repository:
-https://github.com/openml/openml-python.git
-The repository contains instructions for pip installing this package.
- 
\ No newline at end of file
+### Using pip:  
+
+`pip install metalearn`
+
+### From source:
+
+```bash
+git clone https://github.com/byu-dml/metalearn.git
+cd metalearn
+python3 setup.py install
+```
+## Example Usage
+  
+```python
+from metalearn import Metafeatures
+import pandas as pd
+import numpy as np
+
+# X and Y must be a pandas DataFrame and a pandas Series respectively
+X = pd.DataFrame(np.random.rand(8,2))
+Y = pd.Series(['a', 'a', 'a', 'a', 'b', 'b', 'b', 'b'])
+
+metafeatures = Metafeatures()  # instantiate an instance of the base class Metafeatures
+mfs = metafeatures.compute(X, Y)
+```
+
+## Using the Test Suite
```

Comment:
A section on using the tests is great. For now though, let's not include the other sections about openml, benchmarking, etc. until we clean that up.